tests/test_arrow.py

```diff
@@ -1754,6 +1754,11 @@ def test_span_week(self):
         assert floor == datetime(2013, 2, 11, tzinfo=tz.tzutc())
         assert ceil == datetime(2013, 2, 17, 23, 59, 59, 999999, tzinfo=tz.tzutc())
 
+        floor, ceil = self.arrow.span("week", weekday=6)
```

Comment:
We should also test what happens if a bad value is provided (i.e. `week_start=55`)